README.md

```diff
@@ -728,79 +772,94 @@ environments should be exposed as an input variable. For example, the frontend-a
    [terraform init command](https://www.terraform.io/docs/commands/init.html), so the `source` parameter supports the
    exact same syntax as the [module source](https://www.terraform.io/docs/modules/sources.html) parameter, including
    local file paths, Git URLs, and Git URLs with `ref` parameters (useful for checking out a specific tag, commit, or
    branch of Git repo). Terragrunt will download all the code in the repo (i.e. the part before the double-slash `//`) 
-   so that relative paths work correctly. 
-1. Copy all files from the current working directory into the temporary folder. This is useful if you have `.tfvars`
-   files in your local path that you are using to set variables within your templates.
+   so that relative paths work correctly between modules in that repo. 
+1. Copy all files from the current working directory into the temporary folder. This way, Terraform will automatically
+   read in the variables defined in the `terraform.tfvars` file.
 1. Execute whatever Terraform command you specified in that temporary folder. **Note**: if you are passing any file
    paths (other than paths to files in the current working directory) to Terraform via command-line options, those 
    paths must be absolute paths since we will be running Terraform from the temporary folder!
 
-With new approach, copy/paste between environments is minimized. `terraform.tfvars` contains solely the variables 
-that are different between environments and `.terragrunt` specifies from where to download the actual Terraform code.
-To create a new environment, you copy and old one and update just the environment-specific values in `terraform.tfvars`,
-which is about as close to the "essential complexity" of the problem as you can get.
+With new approach, copy/paste between environments is minimized. The `.tfvars` files contain solely the variables 
+that are different between environments. To create a new environment, you copy an old one and update just the 
+environment-specific values in the `.tfvars` files, which is about as close to the "essential complexity" of the 
+problem as you can get.
 
-Just as importantly, since the Terraform code is now defined in a single repo, you can version it (e.g. using Git
+Just as importantly, since the Terraform code is now defined in a single repo, you can version it (e.g., using Git
 tags and referencing them using the `ref` parameter in the `source` URL), and promote a single, immutable version 
-through each environment (e.g. qa -> stage -> prod). This idea is inspired by Kief Morris' blog post [Using Pipelines 
+through each environment (e.g., qa -> stage -> prod). This idea is inspired by Kief Morris' blog post [Using Pipelines 
 to Manage Environments with Infrastructure as 
 Code](https://medium.com/@kief/https-medium-com-kief-using-pipelines-to-manage-environments-with-infrastructure-as-code-b37285a1cbf5).
```

Comment:
Clarified this a bit: https://github.com/gruntwork-io/terragrunt/pull/122/commits/ea0f8cac4f1ac64f7cb41e151da8be8894a14bea